### ISSUE TITLE
Getting std::logic_error

### ISSUE THREAD

**Issue by j-adamczyk** [mir-am/LIBTwinSVM#4]
My code:
```
from time import process_time
from typing import List, Tuple

import numpy as np
from sklearn.datasets import make_classification
from sklearn.metrics import accuracy_score
from sklearn.model_selection import train_test_split

from libtsvm.estimators import TSVM
from sklearn.svm import SVC as sklearn_SVC


def get_dataset():
    # create synthetic dataset
    X, y = make_classification(n_samples=10000, n_features=100,
                               n_informative=90, n_redundant=10,
                               random_state=0)
    X_train, X_test, y_train, y_test = train_test_split(X, y,
                                                        test_size=.2,
                                                        random_state=0)
    return X_train, X_test, y_train, y_test


def test_twin_SVM(X_train: np.ndarray,
                  X_test: np.ndarray,
                  y_train: np.ndarray,
                  y_test: np.ndarray) \
        -> Tuple[List[float], List[float]]:
    clf = TSVM()
    #clf = sklearn_SVC()
    times = []
    sizes = list(range(1000, 10001, 1000))
    accuracies = []

    print("TwinSVM start")
    for size in sizes:
        print(size)
        X_train_smaller = X_train[:size, :]
        y_train_smaller = y_train[:size]

        time1 = process_time()
        clf.fit(X_train_smaller, y_train_smaller)
        time2 = process_time()
        times.append(time2 - time1)

        acc = accuracy_score(y_test, clf.predict(X_test))
        accuracies.append(acc)
    print()

    return times, accuracies

X_train, X_test, y_train, y_test = get_dataset()
twin_times, twin_accuracies = test_twin_SVM(X_train, X_test,
                                                y_train, y_test)
```

Output and error:
```
TwinSVM start
1000

error: index_max(): object has no elements
terminate called after throwing an instance of 'std::logic_error'
  what():  index_max(): object has no elements
```

What is happening, why am I getting this error? Sklearn SVM is working fine, I've also tried it (commented out in the code above).

**Comment by mir-am** [comment]
This error usually happens when the input training samples to `fit` method are problematic.
Nowadays, I'm super busy with my other projects but I'll try your example and investigate the error in the next couple of days.

**Comment by Gapsou** [comment]
I've tried the library as well, I have the exact same error when running the `fit` method. I also have a segmentation fault when combining TSVM with an RBF kernel. Do you have any suggestion ?

**Comment by Gapsou** [comment]
PS: I liked reading your blog :)

**Comment by mir-am** [comment]
> I've tried the library as well, I have the exact same error when running the `fit` method. I also have a segmentation fault when combining TSVM with an RBF kernel. Do you have any suggestion ?

@Glantanamo 
Thanks for using LibTwinSVM.
Please make sure that the `y_train` for the `fit` method only contains a label in `{-1, 1}`. Otherwise, it causes that issue in the optimizer.

**Comment by Gapsou** [comment]
@mir-am Thanks a lot for your response.
If there are more than two classes, how can LIBTwinSVM be used using only values in `{-1, 1}` ?
Thanks again :)

**Comment by mir-am** [comment]
> @mir-am Thanks a lot for your response. If there are more than two classes, how can LIBTwinSVM be used using only values in `{-1, 1}` ? Thanks again :)

This is an example of a multi-class classification using LibTwinSVM:
https://libtwinsvm.readthedocs.io/en/latest/examples/API/mc_clf.html

Note that class labels start at 1 for multi-class classification. i.e., `1,2,3,...`
Also, see the [dataset folder](https://github.com/mir-am/LIBTwinSVM/tree/master/dataset).

**Comment by Gapsou** [comment]
@mir-am Thanks a lot, it works now!
